recipe/meta.yaml

```
@@ -31,7 +31,7 @@ requirements:
     - networkx
     - numpy >=1.18
     - pandas >=1.0
-    - pint >=0.11
+    - pint >=0.18
```

```suggestion
    - pint >=0.11
```
Since this is still `v0.5.2` I would prefer to stick to those requirements unless necessary for python3.10